tests/components/shopping_list/conftest.py

```diff
@@ -19,5 +19,10 @@ def mock_shopping_list_io():
 @pytest.fixture
 async def sl_setup(hass):
     """Set up the shopping list."""
-    assert await async_setup_component(hass, "shopping_list", {})
+
+    entry = MockConfigEntry(domain="shopping_list")
+    entry.add_to_hass(hass)
+
+    assert await hass.config_entries.async_setup(entry.entry_id)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5134240)